CMakeLists.txt

```diff
@@ -446,14 +448,14 @@ endif()
 execute_process(COMMAND mkdir ${MKDIR_OPTS} "${CMAKE_BINARY_DIR}/generated")
 
 # We need to link some packages as dynamic/dependent.
-set(CMAKE_FIND_LIBRARY_SUFFIXES .dll .dylib .so .a)
+set(CMAKE_FIND_LIBRARY_SUFFIXES .lib .dylib .so .a)
 
 # Most dependent packages/libs we want static.
 if(DEFINED ENV{BUILD_LINK_SHARED})
   set(ENV{OSQUERY_BUILD_LINK_SHARED} TRUE)
 endif()
 if(NOT DEFINED ENV{OSQUERY_BUILD_LINK_SHARED})
-  set(CMAKE_FIND_LIBRARY_SUFFIXES .a .dylib .so)
+  set(CMAKE_FIND_LIBRARY_SUFFIXES .lib .a .dylib .so)
```

Comment:
The `OSQUERY_BUILD_LINK_SHARED` is requesting that we link dynamically, so this should have `.dll .lib`